velox/functions/prestosql/InPredicate.cpp

```diff
@@ -293,7 +301,7 @@ class InPredicate : public exec::VectorFunction {
         if (flatArg->isNullAt(row)) {
           boolResult->setNull(row, true);
         } else {
-          bool pass = testFunction(rawValues[row]);
+          bool pass = testFunction(flatArg->valueAt(row));
```

Comment:
rawValues[row] is faster then flatArg->valueAt

Consider adding a separate code path for booleans.

Reply:
Switched to `valueAtFast` which is equivalent to `rawValues_[idx]`